addon/components/drop-zone.js

```diff
@@ -104,9 +127,13 @@ export default Ember.Component.extend({
       }
     }
   },
-
-  config: Ember.computed(function(){
-    let config = this.get('dzConfigHash'),
+  /**
+   * internal config cp
+   * @private
+   * @return {[type]} [description]
+   */
+  _dzConfig: Ember.computed(function(){
+    let config = this.get('confi'),
```

Comment:
This may be the reason the test build is failing. It should be `let config = this.get('config'),` right?